.ci/scripts/shellcheck

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 set -e
-docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:stable "$@"
+docker run --rm -t -v "$PWD:/mnt" -u "$(id -u):$(id -g)" docker.elastic.co/observability-ci/shellcheck "$@"
```

Comment:
Not sure whether we could provide this particular preCommit hook within the apm-pipeline-library somehow. WDYT?

Reply:
I thought it was in the library